tensorflow/python/ops/logging_ops.py

```diff
@@ -190,14 +184,11 @@ def f():
 
     Usage when constructing graphs:
```

Comment:
The best thing to do with 1.x vs. 2.x docstrings is to make a new symbol which is v2-only with the new docstring (no need to mention control dependencies at all there, and certainly no need to mention Sessions) and have a v1-only symbol with the same name which has the old docstring. The v1 symbol should call the v2 symbol.

We're still planning on making a couple 1.x releases I believe, so we should preserve the existing information in the v1 docstring.

Example of this kind of transformation (although it's copying a bit more than you'll need to): https://github.com/tensorflow/tensorflow/commit/dd0b3d3d724f13bb99423e92ea3e32601058ff2a

Note the tf_export lines, which control which versions the symbol is exposed in.

Reply:
@allenlavoie I finished the work as [`dd0b3d3`](https://github.com/tensorflow/tensorflow/commit/dd0b3d3d724f13bb99423e92ea3e32601058ff2a), but do you thing is good to double the code is just for documenting? Or should I leave or close this issue, and re-open when release when finalizing tfv2?

Reply:
A little duplication is fine. They're giving different information.

Reply:
@allenlavoie  I modify as [`b38ef23`](https://github.com/tensorflow/tensorflow/pull/27479/commits/b38ef235d56a340cb080da328f47294ce24a92ec)